dao/src/main/java/org/thingsboard/server/dao/tenant/TenantServiceImpl.java

```diff
@@ -166,6 +169,7 @@ public void deleteTenant(TenantId tenantId) {
         adminSettingsService.deleteAdminSettingsByTenantId(tenantId);
         qrCodeSettingService.deleteByTenantId(tenantId);
         notificationSettingsService.deleteNotificationSettings(tenantId);
+        trendzSettingsService.deleteTrendzSettings(tenantId);
```

Comment:
Please move the deletion of Trendz and notification settings before the `adminSettingsService.deleteAdminSettingsByTenantId`, since they use admin settings API.